source/langtags_schema.json

```diff
@@ -76,11 +81,13 @@
         },
         "localnames": {
           "type": "array",
-          "items": { "type": "string" }
+          "items": { "type": "string" },
+          "uniqueItems": true
         },
         "latnnames": {
           "type": "array",
-          "items": { "type": "string" }
+          "items": { "type": "string" },
+          "uniqueItems": true
```

Comment:
This need not be unique. The key is that each entry aligns with a corresponding entry in the localnames array.